pennylane/math/multi_dispatch.py

```diff
@@ -870,6 +870,50 @@ def norm(tensor, like=None, **kwargs):
     return norm(tensor, **kwargs)
 
 
+@multi_dispatch(argnum=[0])
+def svd(tensor, like=None, **kwargs):
+    """Compute the singular value decomposition of a tensor in each interface.
+
+    For a matrix :math:`A`, the singular value decomposition consist of three matrices :math:`S`, :math:`U` and :math:`Vh`, such that:
+    .. math::
+
+        A = U . Diag(S) . Vh
+
+    Returns:
+        :math:`S`, :math:`U` and :math:`Vh`: full decomposition (if compute_uv is True or None)

```

Comment:
What is `compute_uv`? Might be nice to have an Args and Kwargs section in this docs.

Reply:
It [specifies](https://numpy.org/doc/stable/reference/generated/numpy.linalg.svd.html) computing the `u` and `vh` matrices in addition to the singular values.

Reply:
Thanks!